### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by cveneziani**

### CODE REVIEW

File: _includes/home/en/coworking.html [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      A warmfull space to develop your projects, work, train and share in good mood. Wifi, coffee, tea are included. You just have to show up and start your daily routine. Whether you're coming for 1 hour, 1 day, 1 week, 1 year or all your life... this is as simple as this ;-)
+    </p>
+    <div class="col-left">
+      <div class="block">
+        <h1>Work,<br /> Develop your projects</h1>
+        <p class="first">
+          20 hot desking spots are available. Feel free to choose your spot and meet new coworkers each day.
+        </p>
+        <p>
+          Screens are provided. Some lockers (only for permanent coworkers) are also available.
```
Comment (glibersat) [inline _includes/home/en/coworking.html]: are available if needed.

File: _includes/home/en/events.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<section id="events">
+  <div class="center">Workshops &amp;<br />Events</div>
+  <div class="container">
+    <div class="col-left">
+      <div class="block">
+        <h1>Workshops &amp; talks</h1>
+        <p>
+          If you want to organize a workshop, the meeting room is adjustable to this kind of events. With or without tables, you choose.
```
Comment (glibersat) [inline _includes/home/en/events.html]: , your call.

File: _includes/home/en/the_space.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        <p>
+          A space of 160 square meters with tables, coffee, wifi, meeeting room, relaxation area, kitchen and good mood.
+        </p>
+        <a target="_blank" href="http://www.flickr.com/photos/61299367@N05/sets/72157634949533813/with/9448166157/">See photos</a>
+      </div>
+      <div class="block">
+        <h1>How does it work?</h1>
+        <p>
+          <span>La Coroutine is a collaborative space:</span>
+          Each member participates in the life of the space (washing dishes, water plants, feed the stove, welcoming a newcomer, ...).
```
Comment (glibersat) [inline _includes/home/en/the_space.html]: watering, feeding

File: _includes/home/en/third_place.html [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+          Every monday evening, <a href="http://www.lafermedupommier.com/">Kévin</a>, our market gardener turns the space into a mini organic market. You can pre-order vegetables every weekend for the following week.
+        </p>
+        <p>
+          For lunch, we launched an initiative called <a target="_blank" href="http://wiki.lacoroutine.org/contribution:cocotte">Cocotte</a> : for the most motivated ones, we cook together a vegetarian dish that she share in a warmfull lunch.
+        </p>
+        <p>
+          We also have DIY tools and hardware such as Arduino, soldering iron, drill, etc. that you can use or borrow.
+        </p>
+        <p>
+          In shart, La Coroutine is <strong>your place</strong>! Feel free to contact us if you want to start an activity in the space.
```
Comment (glibersat) [inline _includes/home/en/third_place.html]: In short

File: _includes/home/fr/contact.html [hunk 5]
```diff
@@ -13,7 +13,7 @@
         <h1>Venir</h1>
         <p>
           <span class="adresse">8 rue Molière<br />59000 Lille</span>
-          <span class="horaires">Du Lundi au Vendredi<br /> de 9h00 à 19h+</span>
+          <span class="horaires">Du Lundi au Vendredi<br /> de 9h30 à 19h+</span>
```
Comment (glibersat) [inline _includes/home/fr/contact.html]: ..

**Comment by cveneziani** [comment]
Thanks!

**Pull request merged**